tests/analyse_errors.py

```diff
@@ -0,0 +1,13 @@
+import json
+
+with open('test_data_SanskritLexicalAnalyzer/bg_failing.txt') as f:
+  error = json.load(f)
+
+
+error = error["errors"]
+
+for err in error:
+	x = ' '.join(err['split'])
+	y = err['orig_split']
+	if(x not in y):
```

Comment:
Does this mean that the check for whether the correct split has been found in the generate_bg_Lexical_Analyzer_pass_fail.py (__in_splits) needs to be changed? Maybe it would make sense to update that file itself instead of adding a new one then?

Reply:
You are right. I will do the needful and reopen the PR